README.md

```diff
@@ -37,7 +37,15 @@ Step 3: Install the OS-Agent:
 
 Instructions for installing the OS-Agent can be found [here](https://github.com/home-assistant/os-agent/tree/main#using-home-assistant-supervised-on-debian)
 
-Step 4: Install the Home Assistant Supervised Debian Package:
+Step 4: Install and activate apparmor (needed on raspbian)
```

Comment:
Raspbian is not a supported system by the Home Assistant project.

Reply:
I'm honestly confused.
The docs include following platforms as supported:

raspberrypi
raspberrypi2
raspberrypi3
raspberrypi4
raspberrypi3-64
raspberrypi4-64

Your install instructions run smoothly on Raspbian, as it's a debian distribution, which uses aptitude and dpkg (referenced in the docs).

Can you elaborate why you're stating that you're not supporting this platform although you're documenting that you do?

Reply:
We support running Debian, not it's derivatives (like Raspbian).

This is documented and part of the listed requirements.

Reply:
I understand, but I'm still non the wiser after all, since this didn't explain why though.
You just stated, it is like it is.

Reply:
> I understand, but I'm still non the wiser after all, since this didn't explain why though.

That is a project decision to limit the number of variables and issues. Every operating is a little different (hence your PR, right?), Home Assistant supports and tests a single system. 

You may run anything you like, but that is up to you, don't expect things to work, be documented, tested, or supported.